Tasks/JavaToolInstaller/javatoolinstaller.ts

```diff
@@ -43,12 +43,12 @@ async function getJava(versionSpec: string) {
     if (version) { //This version of Java JDK is already in the cache. Use it instead of downloading again.
         console.log(taskLib.loc('Info_ResolvedToolFromCache', version));
     } else if (fromAzure) { //Download JDK from an Azure blob storage location and extract.
-        console.log(taskLib.loc('RetrievingJdkFromAzure', version));
+        console.log(taskLib.loc('RetrievingJdkFromAzure'));
         compressedFileExtension = getFileEnding(taskLib.getInput('azureCommonVirtualFile', true));
     
         const azureDownloader = new AzureStorageArtifactDownloader(taskLib.getInput('azureResourceManagerEndpoint', true), 
-            taskLib.getInput('azureStorageAccountName', true), taskLib.getInput('azureContainerName', true), taskLib.getInput('azureCommonVirtualFile', false));
-        await azureDownloader.downloadArtifacts(extractLocation, '*' + compressedFileExtension);
+            taskLib.getInput('azureStorageAccountName', true), taskLib.getInput('azureContainerName', true), "");
+        await azureDownloader.downloadArtifacts(extractLocation, '*' + taskLib.getInput('azureCommonVirtualFile', false));
```

Comment:
Is the `'*' +` needed?  ('*.zip' makes sense to me; '*javatool.zip' seems a bit odd.)

Reply:
The Azure library team who owns the downloadArtifacts functionality had suggested using the * in case the entire path was not entered. I can remove it if there are strong feelings against it.

Reply:
Either way is fine by me.  It looks odd to me, but it doesn't look wrong.